web/Dockerfile

```diff
@@ -53,7 +53,7 @@ RUN yum -y install gcc-c++
 # ant for iviewer, make for nodejs
 RUN yum -y install ant make
 RUN yum -y install openssl-devel bzip2-devel expat-devel
-RUN curl -sL https://rpm.nodesource.com/setup | bash
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash
```

Comment:
why don't you use https://rpm.nodesource.com/setup_12.x?

Reply:
Sorry, I didn't even notice the rpm/deb change. I just used the (recommended) URL provided in the warning above.

Reply:
Fixed, thx